sql/plsql/PlSqlParser.g4

```diff
@@ -637,7 +640,19 @@ create_analytic_view
     ;
 
 classification_clause
-    : (CAPTION c=quoted_string)? (DESCRIPTION d=quoted_string)? classification_item*
+// : (CAPTION c=quoted_string)? (DESCRIPTION d=quoted_string)? classification_item*
+// to handle - 'rule contains a closure with at least one alternative that can match an empty string'
+    : caption_clause description_clause? classification_item*
+    | caption_clause? description_clause classification_item*
+    | caption_clause? description_clause? classification_item+
```

Comment:
Probably it makese sense to simplify in the following way:

```suggestion
    : (caption_clause description_clause? | caption_clause? description_clause) classification_item*
    | caption_clause? description_clause? classification_item+
```

Or in the following:

```suggestion
    : caption_clause description_clause? classification_item*
    | caption_clause? (description_clause classification_item* | description_clause? classification_item+)
```